schema/logs/README.md

```diff
@@ -0,0 +1,223 @@
+
+#### Communication
+Includes client / server part of the communication
+
+ **_Inspired by_** :
+ - [ECS - client](https://www.elastic.co/guide/en/ecs/8.6/ecs-client.html)
+ - [ECS - Server](https://www.elastic.co/guide/en/ecs/8.6/ecs-server.html#ecs-server)
+ - [ECS - Source](https://www.elastic.co/guide/en/ecs/8.6/ecs-source.html)
+ - [ECS - Destination](https://www.elastic.co/guide/en/ecs/8.6/ecs-destination.html)
+ - [OTEL - network](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md)
+
+
+ - `sock.family` - Protocol address family which is used for communication. (OTEL driven)
+
+ - `client.domain` -   The domain name of the client system. (ECS driven) Correspond with `source.domain`, (ECS driven)
+ - `client.bytes`  -  Bytes sent from the client to the server. (ECS driven) Correspond with `source.bytes` (ECS driven)
+ - `client.ip` -  IP address of the client (IPv4 or IPv6). (ECS driven) Correspond with `source.ip` (ECS driven)
+ - `client.mac` -  MAC address of the client. (IPv4 or IPv6). (ECS driven) Correspond with `source.mac` (ECS driven)
+ - `client.packets` - Packets sent from the client to the server. (ECS driven) Correspond with `source.packets` (ECS driven)
+
+ - `sock.host.addr` - Local socket address. (OTEL driven) Correspond with `client.address`,`source.address`  (ECS driven)
+ - `sock.host.port` - Local socket port number. (OTEL driven) Corresponds with `client.port`,`source.port` (ECS driven)
+
+ - `server.domain` -   The domain name of the server system. (ECS driven) Correspond with `destination.domain` (ECS driven)
+ - `server.bytes`  -  Bytes sent from the server to the client. (ECS driven)  Correspond with `destination.bytes` (ECS driven)
+ - `server.ip` -  IP address of the server (IPv4 or IPv6). (ECS driven) Correspond with `destination.ip` (ECS driven)
+ - `server.mac` -  MAC address of the server. (IPv4 or IPv6). (ECS driven) Correspond with `destination.mac` (ECS driven)
+ - `server.packets` - Packets sent from the server to the client.. (ECS driven) Correspond with `destination.packets` (ECS driven)
+
+ - `sock.peer.addr` - Remote socket peer address: IPv4 or IPv6 (OTEL driven)  Correspond with `server.address`, `destination.address` (ECS driven)
+ - `sock.peer.name` - Remote socket peer name. (OTEL driven) Correspond with `destination.domain` (ECS driven)
+ - `sock.peer.port` - Remote socket peer port. (OTEL driven)  (OTEL driven) Corresponds with `server.port`,`destination.port` (ECS driven)
+
+---
+
+```text
+
+ ___________________
+ | _______________ |
+ | |XXXXXXXXXXXXX| |
+ | |XXXXXXXXXXXXX| |
+ | |XXXXXXXXXXXXX| |
+ | |XXXXXXXXXXXXX| |
+ | |XXXXXXXXXXXXX| |
+ |_________________|
+     _[_______]_
+  ___[___________]___
+ |         [_____] []|__
+ |         [_____] []|  \__
+ L___________________J      \  \___\/
+  ___________________       /\
+  /###################\    (__)
+
+```
+
+---
+### Setting Up the Logs Mapping
```

Comment:
Overall the doc looks good! Can we add an example log in the RFC (one from the `schema/logs/sample/nginx_access-log.json` looks good)? The example would be great for users to read and paralelly understand schema with an example.